resources/assets/coffee/react/beatmap-discussions/new-reply.coffee

```diff
@@ -74,6 +76,8 @@ class BeatmapDiscussions.NewReply extends React.PureComponent
         className: "#{bn}__footer"
         div className: "#{bn}__actions",
           div className: "#{bn}__actions-group",
+            @renderCancelButton()
```

Comment:
dis coo.

(probably should be removed)

![](https://s.myconan.net/2017-12/firefox_2017-12-18_19-42-36.png)

Reply:
😑